### PR TITLE
[13.0] [MIG] oca_psc_team: Migration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT23196

### CODE REVIEW

**Comment by CarlosRoca13** [comment]
Please review it. @pedrobaeza @Yajo

**Review by pedrobaeza — APPROVED** [review]

File: oca_psc_team/models/project.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class ProjectProject(models.Model):
+    _inherit = ["project.project", "image.mixin"]
+    _name = "project.project"
+
+    short_description = fields.Text(
+        help="Short description to display it in website PSC teams list page."
+    )
+
+    description = fields.Html(
+        help="This will be displayed in the project's detail page of PSC " "team."
```
Comment (yajo) [inline oca_psc_team/models/project.py]:
```suggestion
        help="This will be displayed in the project's detail page of PSC team."
```

**Review by yajo — APPROVED** [review]
Just a minor :wastebasket: black left.

**Comment by CarlosRoca13** [comment]
@Yajo chage done!

**Pull request merged**